Makefile

```diff
@@ -128,13 +129,17 @@ compile:
 	# copy these esm format js files to dist directly.
 	cp -r ${ROOTDIR}/testing/injections ${ROOTDIR}/dist/testing/
 
+.PHONY: compile-documentation-scripts
+compile-documentation-scripts:
+	${ROOTDIR}/node_modules/.bin/tsc --project tsconfig.scripts.json
```

Comment:
sorry I have completely lost track on this. will this just compile without throwing any errors?